schema.graphql

```diff
@@ -191,6 +191,8 @@ enum VolumeType {
     Redeemed
     Collateral
     Staked
+    Locked
+    BridgeVolume
```

Comment:
With this change, I think we will need to create new migration files.

See the section [Workflow - updating the SQL schema](https://github.com/interlay/interbtc-squid#workflow---updating-the-sql-schema).

Reply:
Fixed with commit [13b2881](https://github.com/interlay/interbtc-squid/pull/54/commits/13b2881512930f344e659dcc6b979b91d47edbd6)